test/visual/visual.js

```diff
@@ -60,6 +60,7 @@ module.exports = {
                     userDataDir: `${baselineDir}/${type}/userDataDir`,
                 });
                 page = await browser.newPage();
+                await page.mouse.move(-5, -5);
```

Comment:
just curious, what is the mouse move here for?

Reply:
I should add a comment, but what this does is prevent hover interactions on the UI during the screenshot. At best hovers can cause slight color changes on buttons, and at worse it can actually trigger tooltip a, see the overlay edges story.

Reply:
Added.